packages/react/src/components/Footer/__stories__/Footer.stories.js

```diff
@@ -6,7 +6,6 @@
  */
```

Comment:
@kennylam can stories be added to dotcom shell as well?

Reply:
I was debating this but felt it was redundant so decided against it. But I guess it can't hurt.

Reply:
I think it can help since we can't really see the language selector in the short and micro versions since the open upward and off the screen in the footer storybook story. Thinking about this too, I wonder if we want to vertically center the footer in the footer stories?

Reply:
@jeffchew Yep, I actually did that already as well.

Reply:
hey @kennylam I don't see the change yet.